OpenShift/config_example.sh

```diff
@@ -17,3 +17,7 @@ set -x
 
 # Set loglevel for OpenShift installation
 #LOGLEVEL=debug
+
+# Set Quay credentials for CNV pre-release access
```

Comment:
-1 to storing user and pw in plain text.  Either create the secret ahead of time, gather a token yourself `curl https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/master/tools/token.sh | bash` and inject it, or use the prompt.

Reply:
ack, will create the secret ahead